### PR TITLE
Feature/page indicator

### PR DESCRIPTION
**Pull request by vadymmarkov**
ios simulator screen shot 02 jul 2015 11 09 41

### CODE REVIEW

**Comment by vadymmarkov** [comment]
@zenangst @richardoti Mind having a look.

File: Source/LightboxController.swift [hunk 1]
```diff
@@ -49,12 +55,24 @@ public class LightboxController: UIViewController {
     return layout
     }()
 
+  lazy var pageLabel: UILabel = { [unowned self] in
+    let label = UILabel(frame: CGRectZero)
+    label.setTranslatesAutoresizingMaskIntoConstraints(false)
+    let config = LightboxConfig.sharedInstance.config.pageIndicator
```
Comment (zenangst) [inline Source/LightboxController.swift]: Could you sort these by declarations so that all `let`'s are at the top?

Comment (vadymmarkov) [inline Source/LightboxController.swift]: @zenangst Fixed!

**Comment by zenangst** [comment]
@vadymmarkov looks great, sorry for the delay

**Pull request merged**